rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/TaskanaEngineControllerRestDocumentationTest.java

```diff
@@ -0,0 +1,51 @@
+package pro.taskana.common.rest;
+
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import pro.taskana.common.test.BaseRestDocumentationTest;
+
+class TaskanaEngineControllerRestDocumentationTest extends BaseRestDocumentationTest {
```

Comment:
are we missing a test for the currentVersion endpoint?

Reply:
yes we do. Thanks!